examples/rpi/pte_to_array.py

```diff
@@ -0,0 +1,207 @@
+# Copyright (c) Meta Platforms, Inc. and affiliates.
```

Comment:
use the one we already have from arm. if you want you can make it a devtools utils.